<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorParams</class>
 <widget class="QGroupBox" name="SimulatorParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator Params</string>
  </property>
  <property name="title">
   <string>Simulator Params</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Log File name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="logFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>233</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>233</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max work time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_timeUnits">
       <item>
        <property name="text">
         <string>микросекунды</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>миллисекунды</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>секунды</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>минуты</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>часы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>сутки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>недели</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>месяцы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>года</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
